sys/kern/subr_vmem.c

```diff
@@ -1685,7 +1685,7 @@ vmem_whatis(vmem_addr_t addr, int (*pr)(const char *, ...))
 		}
 		(*pr)("%p is %p+%zu in VMEM '%s' (%s)\n",
 		    (void *)addr, (void *)bt->bt_start,
-		    (vmem_size_t)(addr - bt->bt_start), vm->vm_name,
+		    (vmem_size_t)((char *)addr - (char *)bt->bt_start), vm->vm_name,
```

Comment:
Hmm, why use `caddr_t` instead of `vaddr_t` for `bt->bt_start`?  It seems they should either both be pointers (to get a `ptrdiff_t` result) or both be addresses (to get a `vaddr_t` result)

Reply:
sorry, that was a typo, fixed now.